### ISSUE TITLE
Finish update to ES 7

### ISSUE THREAD

**Issue by sonofmun** [Formulae-Litterae-Chartae/formulae-capitains-nemo#541]
I will keep track here of what needs to be done to finish this change.

- [x] Need to change how corpus level aggregations are calculated. Right now, they depend on the "_type" of the document, which no longer exists in ES 7.

**Comment by sonofmun** [comment]
Aggregations are now correctly calculated after #543 . This is currently being tested on dev.

**Comment by sonofmun** [comment]
These changes have been merged into master and implemented on the server, which has been upgraded to ES 7.12
